src/amber/controller/error.cr

```diff
@@ -33,6 +27,14 @@ module Amber::Controller
       end
     end
 
+    private def internal_server_error_message
+      @ex.inspect_with_backtrace
+    rescue ex : IndexError
+      @ex.message
+    rescue
```

Comment:
Rather than capturing wild card error and printing "An unknown error occured." we should capture wild card and then print both @ex.message and new ex.mesage. Unknown error message is confusing and hides the fact that there is an actual error.